composer.json

```diff
@@ -142,7 +143,7 @@
 			"@ci:php:stan"
 		],
 		"ci:php:cs-fixer": "php-cs-fixer fix --config .php-cs-fixer.php -v --dry-run --diff",
-		"ci:php:lint": "find .*.php *.php Classes Configuration Tests -name '*.php' -print0 | xargs -r -0 -n 1 -P 4 php -l",
+		"ci:php:lint": "parallel-lint --exclude .git --exclude .cache --exclude .Build --exclude typo3temp --exclude Documentation --exclude Resources --exclude .gitlab --exclude .github .",
```

Comment:
I propose we use allowlisting for the file and directories to lint instead of switching to blacklisting (as that would need adjustment every time a tool create a PHP file).

Reply:
parallel-lint does not have whitelist:
https://github.com/php-parallel-lint/PHP-Parallel-Lint?tab=readme-ov-file#command-line-options

Reply:
And combining multiple calls is not what we want, or?

Reply:
I understand this line from the documentation that the last part of the call is the directory to check. (I haven't tried this, and I also have not tried whether we could provide multiple directories here.)